deploy/cr.yaml

```
@@ -14,6 +14,10 @@ spec:
   allowUnsafeConfigurations: false
 #  pause: false
   updateStrategy: SmartUpdate
+  upgradeOptions:
+    versionServiceEndpoint: https://check.percona.com/operator/
+#    apply: 8.0.18-9.3
+    schedule: "* * * * *"
```

let's commit it disabled at this moment
```suggestion
    apply: Disabled
    schedule: "0 2 * * *"
```